_includes/com-ingest/com-ingest-flag-sql-connect-string.md

```diff
@@ -1,6 +1,6 @@
 ### SQL connection strings
 
-| `--driver` | example `--connection-string` | Further information |
+| driver | example connection-string | Further information |
```

Comment:
This is a command line flag, right? doesn't that mean the `--` is needed?

Reply:
I'm going to need to revisit.
The problem with putting `--` without the backticks is the system auto-changes these to an M-dash.
And the reason I changed it is because backticks in headers makes the text impossible to read >.<
Theoretically the quickest fix would be to muck around with CSS. But that's not my final word.

Reply:
Then maybe just don't make it code here? It's better to have it correctly imo than as a code block. Not worth spending a lot of time on :)

Reply:
It works now I've made change to CSS and tested.